cfgov/v1/models/learn_page.py

```diff
@@ -301,20 +304,46 @@ class EventPage(AbstractFilterPage):
     def page_js(self):
         return super(EventPage, self).page_js + ['video-player.js']
 
+    def get_venue_coords(self):
+        # Default to Washington DC coordinates
+        venue_coords = '-77.039628,38.898238'
+
+        if not self.venue_city or not self.venue_state:
+            return venue_coords
+
+        location = '{} {}'.format(self.venue_city, self.venue_state)
+        geocode_api_url = 'https://api.mapbox.com/geocoding/v5/mapbox.places/'
+        geocode_location_url = geocode_api_url + location + '.json'
+
+        params = {'access_token': settings.MAPBOX_ACCESS_TOKEN}
+        response = requests.get(geocode_location_url, params=params)
+
+        if response.status_code != 200:
+            return venue_coords
+
+        try:
+            geo_data = json.loads(response.text)
```

Comment:
You can simplify this by using the built-in [`response.json()`](https://2.python-requests.org//en/master/user/quickstart/#json-response-content) method.